test/data/register-a-death-files.yml

```diff
@@ -16,7 +16,7 @@ lib/smart_answer_flows/register-a-death/questions/where_did_the_death_happen.gov
 lib/smart_answer_flows/register-a-death/questions/which_country.govspeak.erb: e816742682e96117df80d67faff13fc8
 lib/smart_answer_flows/register-a-death/questions/which_country_are_you_in_now.govspeak.erb: 9c9399d1588c7d5e01ebf1bf5efa31a1
 lib/smart_answer_flows/register-a-death/register_a_death.govspeak.erb: 1820ba3a5f3ed47816c7f151217be439
-lib/smart_answer/calculators/register_a_death_calculator.rb: 8483d0017c4a653602ff3dc09233ddc4
+lib/smart_answer/calculators/register_a_death_calculator.rb: 003375345f9c8e37280c6e4700383795
 lib/data/rates/register_a_death.yml: f24583d68edd4b06242b5a92e08e25c3
 lib/smart_answer_flows/shared/_overseas_passports_embassies.govspeak.erb: 1d83fae34e0ee72ce1c1554bdb766d21
 lib/smart_answer_flows/shared/births_and_deaths_registration/_button.govspeak.erb: bdd3818dfd2b9d6396daf6f60fd887a0
```

Comment:
I was surprised not to see a change in the checksum for `lib/data/registrations.yml` in this diff. However, it looks as if it isn't included in the checksums at all. I think it should probably be included, shouldn't it?